lib/p2p/Pool.ts

```diff
@@ -32,6 +31,10 @@ class Pool extends EventEmitter {
     this.bindServer(this.server);
   }
 
+  get peerCount(): number {
+    return Object.keys(this.peers).length;
```

Comment:
This returns a fixed value of `5`, in other words the number of members in `PeerList`. Since `peers` (on PeerList) is private, we'll need to add some public logic in that class to return the number of peers. I also think it's worthwhile adding in some check to make sure we only count connected peers.